### PR TITLE
update ga id to use one created by ds

### PR DESCRIPTION
**Pull request by aekong**

### CODE REVIEW

File: config/google_analytics.settings.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-account: UA-120925231-1
+account: UA-122224372-11
```
Comment (jszwedko) [inline config/google_analytics.settings.yml]: Is this in the digital services GA account? I'm not seeing it as a property.

Comment (jszwedko) [reply config/google_analytics.settings.yml]: Or rather, did we create a new one for this site? Is it possible to just add it as a property to the existing GA account (102831134)? cc/ @hshaosf who set up the general Digital Services GA account.

Comment (aekong) [reply config/google_analytics.settings.yml]: yes, a new one was created.  i didn't know we had one already 😨

Comment (hshaosf) [reply config/google_analytics.settings.yml]: I think there are a couple of GA accounts. There is also one that's being used for the D7 sites.

Comment (hshaosf) [reply config/google_analytics.settings.yml]: probably outside of this PR, but i would recommend taking a look at implementing Google Tag Manager (https://www.google.com/analytics/tag-manager/ ) versus vanilla GA.

Comment (jszwedko) [reply config/google_analytics.settings.yml]: @aekong I added you to the existing DS GA account.

It'd be nice to have them all in one place (imo).

Comment (aekong) [reply config/google_analytics.settings.yml]: what if this is just for dev/testing?

Comment (aekong) [reply config/google_analytics.settings.yml]: going to look into ga tag manager per @hshaosf's suggestion

Comment (jszwedko) [reply config/google_analytics.settings.yml]: +1 for Google Tag Manager. Either way I think further proliferation of Google Analytics accounts makes it harder to keep track of and administer them.

**Comment by aekong** [comment]
closing this pull request per discussions above

**Pull request closed without merging**